scripts/post_deploy_tdr.py

```diff
@@ -40,11 +44,14 @@ def verify_sources():
     }
     assert tdr_catalogs, tdr_catalogs
     futures = []
+    all_sources = set()
     with ThreadPoolExecutor(max_workers=16) as tpe:
-        for source in set(chain.from_iterable(map(config.sources, tdr_catalogs))):
-            source = TDRSourceSpec.parse(source)
-            for check in (tdr.check_api_access, tdr.check_bigquery_access, verify_source):
-                futures.append(tpe.submit(check, source))
+        for catalog in tdr_catalogs:
+            catalog_sources = config.sources(catalog)
+            for source in catalog_sources - all_sources:
+                source = TDRSourceSpec.parse(source)
+                futures.append(tpe.submit(verify_source, catalog, source))
```

Comment:
AFAICT, the removed line was the only call to `check_api_access` so we should remove that now unused method, but maybe retain the log call it makes.